core/domain/classifier_services.py

```diff
@@ -153,6 +153,44 @@ def handle_non_retrainable_states(exploration, state_names, exp_versions_diff):
         job_exploration_mappings)
 
 
+def convert_strings_to_float_numbers_in_classifier_data(classifier_data):
```

Comment:
See comment on other PR about naming the fields that specifically need conversion (more deterministic)

Reply:
Done. Added a more partially deterministic decrypt approach.